packages/commercetools/theme/config.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  api: {
+    uri: 'https://api.commercetools.com/vsf-ct-dev/graphql',
+    authHost: 'https://auth.sphere.io',
+    projectKey: 'vsf-ct-dev',
+    clientId: 'xlea3xo3vcavMN5kmDlFP4nu',
+    clientSecret: 'JejrKtQgU_KkNxPn_96UEAaEoPocNFqy',
+    scopes: [
+      'create_anonymous_token:vsf-ct-dev',
+      'manage_my_orders:vsf-ct-dev',
+      'manage_my_profile:vsf-ct-dev',
+      'manage_my_shopping_lists:vsf-ct-dev',
+      'manage_my_payments:vsf-ct-dev',
+      'view_products:vsf-ct-dev',
+      'view_published_products:vsf-ct-dev'
+    ],
+  },
+  locale: 'en',
+  currency: 'USD',
+  country: 'US',
+  countries: [
+    { name: 'US', label: 'United States' },
+    { name: 'AT', label: 'Austria' },
+    { name: 'DE', label: 'Germany' },
+    { name: 'NL', label: 'Netherlands' },
+  ],
+  currencies: [
+    { name: 'EUR', label: 'Euro' },
+    { name: 'USD', label: 'Dollar' },
```

Comment:
IMO it's better to use just "EUR" and "USD" in this case, or you should rename it to "American Dollar"